src/frontend/src/lib/styles/global/button.scss

```diff
@@ -59,15 +59,15 @@ button {
 	}
 
 	&.secondary {
-		background: var(--color-cetacean-blue);
-		color: var(--color-white);
-	}
-
-	&.tertiary {
 		background: var(--color-alice-blue);
 		color: var(--color-primary);
 	}
 
+	&.secondary-alt {
```

Comment:
A comment "TODO: to be removed"?

Reply:
i will put it as "TODO: check with design team if it is stiil in use", because for what I undersstood we will have some usecase